setup/blacklist.sh

```diff
@@ -18,6 +18,5 @@ cp conf/iptables-persistent /etc/init.d/iptables-persistent
 ipset create blacklist hash:net
 iptables -I INPUT -m set --match-set blacklist src -j DROP
 time /etc/cron.daily/blacklist
-source setup/dialog.sh
```

Comment:
Removed optional GeoIP blocking, but I am working on that feature more in my own IPSet list